README.md

```diff
@@ -4,32 +4,65 @@
+
+##### RiakCS
+
+It is recommended to use [RIAK-CS](http://docs.basho.com/riak/cs/latest) only for large scale or industry deployments.
+Follow [this](http://docs.basho.com/riak/cs/2.1.1/tutorials/fast-track/local-testing-environment/) tutorial for installation of RiakCS.
+
+
+###### Configuration
+After successful installation, the user’s access key and secret key are returned in the `key_id` and `key_secret` fields respectively. Use these keys to update **key** and **secret** your config file _( storage.conf.example )_
+
+Holmes-Storage uses Amazon S3 signature version 4 for authentication. To enable authV4 on riak-cs, add `{auth_v4_enabled, true}` to advanced.config file ( should be in `/riak-cs/etc/`)
+
+##### Fake-S3
+
+We recommend [Fake-S3](https://github.com/jubos/fake-s3) as a simple starting point for exploring the system functionality. Most of the current developers of Holmes-Storage are using [Fake-S3](https://github.com/jubos/fake-s3) for quick testing purposes. [Minio](https://github.com/minio/minio) is also encouraged if you want to engage more with development and do more testing.
+
+
+###### Configuration
+
+Check out the documentation of [Fake-S3](https://github.com/jubos/fake-s3) on how to install and run it. Afterwards, go to `/config/storage.conf` of Holmes-Storage and set the IP and Port your ObjectStorage server is running on. You can decide whether you want your Holmes client to send HTTP or HTTPS requests to the server through the `Secure` parameter.
+
+#### II. Document Stores
+We support two primary object storage databases. 
 - Cassandra
 - MongoDB
 
+We recommend a Cassandra cluster for large deployments.
+
 ##### Cassandra 
-Holmes-Storage supports a single or cluster installation of Cassandra version 3.5.x and higher. The version requirement is because of the significant improvement in system performance when leveraging the newly introduced [SASIIndex](https://github.com/apache/cassandra/blob/trunk/doc/SASI.md) for secondary indexing. We highly recommend deploying Cassandra as a cluster with a minimum of three Cassandra nodes in production environments.
+Holmes-Storage supports single node or cluster installation of Cassandra version 3.5.x and higher. The version requirement is because of the significant improvement in system performance when leveraging the newly introduced [SASIIndex](https://github.com/apache/cassandra/blob/trunk/doc/SASI.md) for secondary indexing. We highly recommend deploying Cassandra as a cluster with a minimum of three Cassandra nodes in production environments.
```

Comment:
we should bump this up to 3.10. Cassandra did a lot of bug fixes for sasi and material views. 3.5 works...3.10 works a lot better.

leveraging the newly introduced [SASIIndex](https://github.com/apache/cassandra/blob/trunk/doc/SASI.md) for secondary indexing. => leveraging the newly introduced [SASIIndex](https://github.com/apache/cassandra/blob/trunk/doc/SASI.md) and Material Views [need a link]